src/index.ts

```diff
@@ -167,20 +169,35 @@ function barfTurn() {
     // d. get dressed
     meatOutfit(embezzlerUp, [], underwater);
 
-    adventureMacroAuto(
-      location,
-      Macro.externalIf(
-        underwater,
+    if (
+      !embezzlerUp &&
+      myInebriety() > inebrietyLimit() &&
+      globalOptions.ascending &&
+      estimatedTurns() - digitizedMonstersRemaining() <=
+        availableAmount($item`Map to Safety Shelter Grimace Prime`)
+    ) {
+      const choiceToSet =
+        availableAmount($item`distention pill`) <
+        availableAmount($item`synthetic dog hair pill`) +
+          availableAmount($item`Map to Safety Shelter Grimace Prime`)
```

Comment:
Why add `availableAmount($item'Map to Safety Shelter Grimace Prime')`? Isn't just a less than comparison sufficient?

Reply:
I was modelling the logic on something bean did, to avoid constantly toggling the property. Let me reflect and make sure taht this makes sense.